### PR TITLE
Extending: more edits

### PR DESCRIPTION
**Pull request by mejiaj**
- Spacing fixes
- Use correct component path
- Small copy edits

### CODE REVIEW

File: _includes/sxs/02-extending/part-three.html [hunk 1]
```diff
@@ -93,12 +93,12 @@ <h3 class="step">Add stylesheet partial.</h3>
     <h4 class="filename"><span class="material-icons" aria-hidden="true">description</span> _theme/_uswds-theme-custom-styles.scss</h4>
 <pre class="blur">@use "uswds-core" as *;
 @forward "../assets/css/styles.scss";
-@forward "../assets/css/_my-card.scss"; 
-</pre><pre class="add">@forward "../assets/css/_my-testimonial.scss";</pre>
+@forward "../assets/css/_my-card.scss";
+</pre><pre class="add">@forward "../assets/css/components/_my-testimonial.scss";</pre>
```
Comment (mahoneycm) [inline _includes/sxs/02-extending/part-three.html:97]: We changed the instructions to create the components dir in `_theme` 

```suggestion
@forward "../_theme/components/_my-card.scss";
</pre><pre class="add">@forward "../_theme/components/_my-testimonial.scss";</pre>
```

File: _includes/sxs/02-extending/part-two.html [hunk 2]
```diff
@@ -50,19 +50,19 @@ <h3 class="step">Import the new stylesheet into USWDS custom styles.</h3>
   <div class="editor">
     <h4 class="filename"><span class="material-icons" aria-hidden="true">description</span> _theme/_uswds-theme-custom-styles.scss</h4>
 <pre class="blur">@use "uswds-core" as *;
-@forward "../assets/css/styles.scss";  
+@forward "../assets/css/styles.scss";
 </pre><pre class="add">@forward "../assets/css/_my-card.scss";</pre>
```
Comment (mahoneycm) [inline _includes/sxs/02-extending/part-two.html:54]: Additional `_theme/components` change

```suggestion
</pre><pre class="add">@forward "../_theme/components/_my-card.scss";</pre>
```

**Review by mahoneycm — CHANGES_REQUESTED** [review]
Two changes for the `_theme/components` dir then we should be good 👍

**Comment by mejiaj** [comment]
Committed your suggestions @mahoneycm.

**Pull request merged**